cmd/main.go

```diff
@@ -66,22 +69,25 @@ func handle() {
 
 	gceDriver := driver.GetGCEDriver()
 
-	//Initialize GCE Driver (Move setup to main?)
-	cloudProvider, err := gce.CreateCloudProvider(vendorVersion, *gceConfigFilePath)
+	// Initialize GCE Driver (Move setup to main?)
+	cloudProvider, err := gce.CreateCloudProvider(vendorVersion, *cloudConfigFilePath)
 	if err != nil {
 		klog.Fatalf("Failed to get cloud provider: %v", err)
 	}
 
 	mounter := mountmanager.NewSafeMounter()
```

Comment:
i dont think we need things like the mounter, deviceUtils, or statter if we're not running the node service